src/count/main.js

```diff
@@ -0,0 +1,60 @@
+const config = require('./src/javascripts/config');
+const oCommentApi = require('o-comment-api');
+const defaultConfig = require('./config.js');
+const oCommentUtilities = require('o-comment-utilities');
+const Widget = require('./src/javascripts/widget');
+
+/**
+ * Default config (prod) is set.
+ */
+config.set(defaultConfig);
+
+/**
+ * Enable data caching.
+ */
+oCommentApi.setConfig('cache', true);
+
+
+/**
+ * Widget.js exposed as main constructor
+ * @type {object}
+ */
+module.exports = Widget;
+
+/**
+ * Adds or overrides configuration options.
+ *
+ * @param  {string|object} keyOrObject Key or actually an object with key-value pairs.
+ * @param  {anything} value Optional. Should be specified only if keyOrObject is actually a key (string).
+ * @return {undefined}
+ */
+module.exports.setConfig = function () {
+	config.set.apply(this, arguments);
+};
+
+module.exports.utilities = oCommentUtilities;
```

Comment:
It looks like non of the count exports are used or made public? (except the default `module.exports = Widget;`)

Reply:
I wondered about this! To be completely honest, I'm a bit confused about where these exports are being used. I can't find it in `next-article` or any of the `alphaville` apps. Can they just be removed?

Reply:
seems so! 🔥